### PR TITLE
Typo and summarize plugwise changes

### PR DESCRIPTION
**Pull request by CoMPaTech**
## Proposed change
Spotted a typo and suggestions for summarizing the Plugwise changes (while also creditting co-author).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2020-06-03-release-111.markdown [hunk 1]
```diff
@@ -107,11 +110,8 @@ continue, resulting in a faster startup.
...
 - Add lock platform to ozw component ([@mrk-its] - [#36103]) ([ozw docs]) (new-platform)
 - Add fan platform to ozw component ([@cgarwood] - [#35249]) ([ozw docs]) (new-platform)
-- Update plugwise to async and config_flow ([@CoMPaTech] - [#33691]) ([plugwise docs]) (breaking-change) (new-platform)
-- Update plugwise to async and config_flow sensor part ([@CoMPaTech] - [#36219]) ([plugwise docs]) (new-platform)
+- Update plugwise to async and config_flow, adding binary_sensor, sensor and switch ([@CoMPaTech], [@bouwew] - [#33691], [#36219], [#36378], [#36383]) ([plugwise docs]) (breaking-change) (new-platform)
 - Refactor Synology entries to allow not fetching the API when it's disabled + add security binary sensor ([@Quentame] - [#35565]) ([synology_dsm docs]) (breaking-change) (new-platform)
-- Update plugwise to async and config_flow binary_sensor part ([@CoMPaTech] - [#36378]) ([plugwise docs]) (new-platform)
-- Update plugwise to async and config_flow switch part ([@CoMPaTech] - [#36383]) ([plugwise docs]) (new-platform)
 - Add climate platform to Insteon ([@teharris1] - [#35763]) ([insteon docs]) (new-platform)
```
Comment (frenck) [inline source/_posts/2020-06-03-release-111.markdown]: These are commits and generated, please revert this change.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_posts/2020-06-03-release-111.markdown [hunk 2]
```diff
@@ -83,13 +83,16 @@ continue, resulting in a faster startup.
...
   and locks is added this release!
-- [@gadgetmobile] when all out on the Blebox integration, adding support for
+- [@gadgetmobile] went all out on the Blebox integration, adding support for
   a lot of platforms!
 - Google Assistant now supports using a select helper (aka `input_select`),
   amazing work [@ZephireNZ]!
 - [@frenck] added two new built-in Home Assistant events, helpful for
   automations: `automation_reloaded` and `scene_reloaded`. Using this as a
   trigger can be used for, e.g., re-applying a scene when it was changed.
+- The Plugwise integration has been improved by [@bouwew] and [@CoMPaTech],
```
Comment (frenck) [inline source/_posts/2020-06-03-release-111.markdown:93]: The user bouwew isn't listed in any contribution, hence it cannot be linked.

File: source/_posts/2020-06-03-release-111.markdown (hunk 1 above)
Comment (CoMPaTech) [reply source/_posts/2020-06-03-release-111.markdown]: Reverting, makes sense why Martin is always making sure the titles are optimal :)

File: source/_posts/2020-06-03-release-111.markdown (hunk 2 above)
Comment (CoMPaTech) [reply source/_posts/2020-06-03-release-111.markdown:93]: It isn't allowed to attribute credit where due manually? (From the above comment fully understood - and maybe I'd need to revert the part mentioning him and summarizing the PRs in breaking as well then?)

Comment (frenck) [reply source/_posts/2020-06-03-release-111.markdown:93]: Sure, but it isn't linking now 😉 

Add `[@bouwew]: https://github.com/bouwew` to that same paragraph (at the end)

File: source/_posts/2020-06-03-release-111.markdown [hunk 3]
```diff
@@ -838,6 +841,7 @@ The integrations below have been removed:
 [@bdraco]: https://github.com/bdraco
 [@bieniu]: https://github.com/bieniu
 [@boralyl]: https://github.com/boralyl
+[@bouwew]: https://github.com/bouwew
```
Comment (frenck) [inline source/_posts/2020-06-03-release-111.markdown]: This is generated. Please move it to the same paragraph (locally within the paragraph) as it is used (at the end), as I mentioned in my initial comment.

Comment (CoMPaTech) [reply source/_posts/2020-06-03-release-111.markdown]: Quickly installed Jekyll to ensure it was rendering ok, I hope it's correct now (it renders correctly locally). Apologies and will fetch more coffee next commit round.

**Review by frenck — APPROVED** [review]
Thanks, @CoMPaTech 👍
